### PR TITLE
test: fix failure for swagger 2 go clients 

### PR DESCRIPTION
**Pull request by zepatrik**
## Checklist

- [ ] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md)
      and signed the CLA.
- [ ] I have read the [security policy](../security/policy).
- [ ] I confirm that this pull request does not address a security
      vulnerability. If this pull request addresses a security vulnerability, I
      confirm that I got green light (please contact
      [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push
      the changes.
- [ ] I have added tests that prove my fix is effective or that my feature
      works.
- [ ] I have added necessary documentation within the code base (if
      appropriate).

### CODE REVIEW

File: scripts/test.sh [hunk 1]
```diff
@@ -45,6 +45,10 @@ golang () {
   echo "Testing Golang..."
 
   dir="clients/${PROJECT}/go"
+  if ! [ -f README.md ]; then
+    echo "package main" > "$dir/main.go"
+    echo "func main(){}" >> "$dir/main.go"
```
Comment (aeneasr) [inline scripts/test.sh:50]: How about

```
(cd "${dir}/client" && go mod tidy && go build -o "$(mktemp)" .)
(cd "${dir}/models" && go mod tidy && go build -o "$(mktemp)" .)
```

instead?

**Review by aeneasr — CHANGES_REQUESTED** [review]

File: scripts/test.sh (hunk 1 above)
Comment (aeneasr) [reply scripts/test.sh:50]: I don't understand the question?

Your patch doesn't test the build, it just tests an empty function. My suggestion builds client and models directories so we test the build actually (if the code is buildable)

Comment (zepatrik) [reply scripts/test.sh:50]: Will there only ever be those two directories, or could we have more that are then missed by that test?

Comment (aeneasr) [reply scripts/test.sh:50]: only directories for swagger generated specs (not openapi3)

**Pull request merged**